src/components/app-content-renderer/styles/panels.scss

```diff
@@ -149,6 +149,12 @@
 
   }
 
+  .ins-c-third--panel-wrapper {
+    background: url(https://raw.githubusercontent.com/RedHatInsights/frontend-assets/master/src/background-images/new-landing-page/ctl-double_gradient.svg);
```

Comment:
Hey can we use https://cloud.redhat.com/apps/frontend-assets/background-images/new-landing-page/ctl-double_gradient.svg instead of relying on github for this file?